File: navigation/navigation-safe-args-generator/src/test/test-data/expected/nav_generator_test/java/nested_overridden_action/SettingsFragmentDirections.java
package foo.flavor;

import androidx.annotation.CheckResult;
import androidx.annotation.NonNull;
import foo.SettingsDirections;

public class SettingsFragmentDirections {
    private SettingsFragmentDirections() {
    }

    @CheckResult
    @NonNull
    public static SettingsDirections.Main main() {
        return SettingsDirections.main();
    }

    @CheckResult
    @NonNull
    public static SettingsDirections.Exit exit() {
        return SettingsDirections.exit();
    }
}

File: navigation/navigation-safe-args-generator/src/test/test-data/expected/nav_generator_test/kotlin/nested_overridden_action/SettingsFragmentDirections.kt
package foo.flavor

import androidx.`annotation`.CheckResult
import androidx.navigation.NavDirections
import foo.SettingsDirections
import kotlin.String

public class SettingsFragmentDirections private constructor() {
  public companion object {
    @CheckResult
    public fun main(enterReason: String = "DEFAULT"): NavDirections =
        SettingsDirections.main(enterReason)

    @CheckResult
    public fun exit(exitReason: String = "DEFAULT"): NavDirections =
        SettingsDirections.exit(exitReason)
  }
}
